src/assets/content/dictionary/wireframe.md

```diff
@@ -1,6 +1,10 @@
 ---
 title: wireframe
-definition: ''
-perspectives: []
+definition: 'A wireframe is a two-dimensional skeletal outline of a webpage or app. Wireframes provide a clear overview of the page structure, layout, information architecture, user flow, functionality, and intended behaviors.'
+sources:
+- sourceurl: https://careerfoundry.com/en/blog/ux-design/what-is-a-wireframe-guide/
+perspectives:
+- meaning: is tool to get organized and visualize a project before I start writing code
```

Comment:
```suggestion
- meaning: a tool to get organized and visualize a project before I start writing code
```
suggestion: I updated the meaning so that the strings will concat with proper grammar.